### DISCUSSION TITLE
new rabbit theme

### DISCUSSION THREAD

**Discussion by sayol** [JanDeDobbeleer/oh-my-posh#5768] [Themes]
[rabbit.omp.json](https://github.com/user-attachments/files/17483639/rabbit.omp.json)

**Comment by vaimalaviya1233** [comment]
![image](https://github.com/user-attachments/assets/fe83329c-7170-4e20-a5e7-a242b8e98ce6)
 it would also be better to see picture with theme file :)

**Comment by vaimalaviya1233** [comment]
theme looks nice
